public/employers/tesla.yml

```diff
@@ -89,4 +89,14 @@ citations:
         date: 2020-04-07T16:35:44-04:00
       - source: Bloomberg
         link: https://www.bloomberg.com/news/articles/2020-04-17/hospitals-made-use-of-modified-tesla-donated-breathing-machines
-        date: 2020-04-17T12:30-07:00
+        date: 2020-04-17T12:30:00-07:00
```

Comment:
The mayor of Fremont did allowed the factory to open up, so I think a -2 may be overkill.

Reply:
Source? That's good info.

Reply:
Added that source and updated the description.